thirdparty/hnswlib/hnswlib/hnswalg.h

```diff
@@ -1157,7 +1157,7 @@ class HierarchicalNSW : public AlgorithmInterface<dist_t> {
         }
 
         tableint currObj = enterpoint_node_;
-        auto vec_hash = knowhere::hash_vec((const float*)query_data, *(size_t*)dist_func_param_);
+        // auto vec_hash = knowhere::hash_vec((const float*)query_data, *(size_t*)dist_func_param_);
```

Comment:
is this intentional?

Reply:
yes, Zihao will update knowhere::hash_vec() to support binary data later